src/golang/config/config.go

```diff
@@ -3,16 +3,19 @@ package config
 import (
 	"io/ioutil"
 	"os"
+	"path"
 
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
 	log "github.com/sirupsen/logrus"
 	"gopkg.in/yaml.v2"
 )
 
 type ServerConfiguration struct {
-	AqPath             string `yaml:"aqPath" json:"aq_path"`
-	EncryptionKey      string `yaml:"encryptionKey" json:"encryption_key"`
-	RetentionJobPeriod string `yaml:"retentionJobPeriod"`
-	ApiKey             string `yaml:"apiKey"`
+	AqPath             string                `yaml:"aqPath" json:"aq_path"`
```

Comment:
Why do we still keep `aqPath` here? Should the storage be parsed from `storageConfig` now?

Reply:
If the storage is S3 we would not be able to determine `aqPath` from the `storageConfig`, since there would be no local filepath.

Reply:
Thanks! Actually I think I missed something, `aqPath` will be used in places other than storage so it does make sense to separate that from storage. This also resolved the above question.